src/sources/webgl.test.ts

```diff
@@ -1,15 +1,8 @@
-import { isHeadlessChrome } from '../../tests/utils'
-import { BotdError } from '../types'
 import getWebGL from './webgl'
 
 describe('Sources', () => {
   describe('webgl', () => {
     it('returns expected values or throws', () => {
-      if (isHeadlessChrome()) {
-        expect(() => getWebGL()).toThrow(new BotdError(-4, 'WebGLRenderingContext is null'))
-        return
-      }
```

Comment:
This removal is excessive, headless Chrome still behaves differently in this source

Reply:
fixed